src/server/services/buurt/datasets.ts

```diff
@@ -583,6 +650,27 @@ function transformMeldingDetailResponse(
   };
 }
 
+function transformLaadpalenDetailResponse(
```

Comment:
Ik realiseer me nu dat die `if (!detailUrl) url = listUrl` constructie echt super inefficient is. Ik ga hier wat anders op verzinnen.